### PR TITLE
Allow for Metacard Actions in a Dropdown

### LINKED ISSUE #275: Allow Index Card Actions in ellipsis dropdown

### PR DESCRIPTION
**Pull request by willwill96**
image

fixes #275

### CODE REVIEW

**Comment by willwill96** [comment]
Need to move the menu to the other side of the index card

File: src/main/webapp/components/index-cards/actions-dropdown.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      onClick={e => {
+        e.stopPropagation()
+        e.preventDefault()
+      }}
+    >
+      <IconButton onClick={open}>
+        <MoreVertIcon />
+      </IconButton>
+      <Menu anchorEl={anchorEl} open={isOpen} onClose={close}>
+        {/* Inject onFinished prop to children so that children know when to close menu */}
```
Comment (willwill96) [inline src/main/webapp/components/index-cards/actions-dropdown.js]: Maybe there's a better way to do this?

**Comment by willwill96** [comment]
Need to figure out a way to keep the idea of a metacard interaction and standard card action separate

**Comment by willwill96** [comment]
Leaving out workspaces until that file has been migrated
Left in the card actions as proof that they can work in tandem with the dropdown

**Comment by gjvera** [comment]
What's the difference between clicking `Edit Form` and clicking on the actual index card?

**Comment by willwill96** [comment]
@gjvera There is no difference. It's there in case the user doesn't know what clicking the index card will do

**Pull request merged**